version: "3.7"

services:
  jupyter:
    image: jupyter/datascience-notebook
    user: root
    volumes:
      - jupyter:/home/jovyan/work:rw
    networks:
      - main
    environment:
      - GRANT_SUDO=yes

volumes:
  jupyter:

networks:
  main:
    external: true
    name: Main_default
